contrib/format-ltsv/pom.xml

```diff
@@ -20,16 +20,16 @@
 -->
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
   <modelVersion>4.0.0</modelVersion>
-
+  
   <parent>
     <artifactId>drill-contrib-parent</artifactId>
     <groupId>org.apache.drill.contrib</groupId>
     <version>1.16.0-SNAPSHOT</version>
   </parent>
-
+  
   <artifactId>drill-format-ltsv</artifactId>
   <name>contrib/ltsv-format-plugin</name>
-
+  
```

Comment:
please remove these extra white spaces here and above.

Reply:
Done.